python/lsst/sims/catalogs/db/dbConnection.py

```diff
@@ -155,8 +155,12 @@ def _connect_to_engine(self):
 
         #DbAuth will not look up hosts that are None, '' or 0
         if self._host:
-            auth = DbAuth(
-                    os.path.join(os.environ["HOME"], ".lsst", "db-auth.yaml"))
+            # This is triggered when you need to connect to a remote database.
+            # Use 'HOME' as the default location (backwards compatibility) but fail graciously
+            authdir = os.getenv('HOME')
+            if os.getenv('HOME') is None:
+                authdir = getPackageDir('SIMS_CATALOGS')
```

Comment:
```suggestion
            if authdir is None:
                # Use an empty file in this package, which causes
                # a fallback to database-native authentication.
                authdir = getPackageDir('sims_catalogs')
```

Reply:
Note that while package directory is in an uppercase environment variable, the package name itself and hence the standard way to call `getPackageDir()` uses lowercase.